assets/scripts/dialogs/UpgradeDialog.scss

```diff
@@ -12,15 +11,76 @@
     [dir="rtl"] & {
       text-align: right;
     }
-  }
 
-  .dialog-content {
-    overflow: hidden;
-    overflow-y: scroll;
+    [role="subtitle"] {
+      color: #1b914e;
```

Comment:
We'd like use defined UI colors as much as possible. Here is a [visual guide to UI colors](https://streetmix.readthedocs.io/en/latest/contributing/ui/colors/), and the SCSS variables are defined here [in this file](https://github.com/streetmix/streetmix/blob/main/assets/styles/_variables.scss).

To refer to UI colors, add this line to the top of the SCSS file:

```scss
@import "../../styles/variables.scss";
```

And then refer to the color closest to the one you want, e.g.

```scss
[role="subtitle"] {
  color: $colour-emerald-500;
}
```

We can do the same for the border colors, below.